src/current/v23.1/how-does-an-enterprise-changefeed-work.md

```diff
@@ -15,6 +15,10 @@ With [`resolved`]({% link {{ page.version.version }}/create-changefeed.md %}#res
 
 As rows are updated, added, and deleted in the targeted table(s), the node sends the row changes through the [rangefeed mechanism]({% link {{ page.version.version }}/create-and-configure-changefeeds.md %}#enable-rangefeeds) to the changefeed encoder, which encodes these changes into the [final message format]({% link {{ page.version.version }}/changefeed-messages.md %}#responses). The message is emitted from the encoder to the sink—it can emit to any endpoint in the sink. In the diagram example, this means that the messages can emit to any Kafka Broker.
 
+{{site.data.alerts.callout_info}}
+When you create a changefeed using [change data capture queries]({% link {{ page.version.version }}/cdc-queries.md %}), the [optimizer]({% link {{ page.version.version }}/cost-based-optimizer.md %}) will evaluate and optimize the query before the job sends the selected row changes to the changefeed encoder. As a result, changefeeds with CDC queries encode only the filtered messages, which helps improve efficiency of the job.
```

Comment:
@amruss does this make sense here?

Reply:
As a result, changefeeds with CDC queries encode only the filtered messages, which helps improve efficiency of the job.
- this is true with or without the optimizer 
- I think you should add since the optimizer can restrict which rows we consider during scans, inital scans and catch up scans can be much more efficient using CDC queries